ksa_compliance/background_jobs.py

```
@@ -14,18 +14,24 @@
 
 
 @frappe.whitelist()
-def add_batch_to_background_queue(check_date=datetime.date.today()):
+def add_batch_to_background_queue(check_date: datetime.date = datetime.date.today()):
```

Ruff (0.15.10)

[warning] Do not perform function call `datetime.date.today` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)